niceml/mkdocs/mdop.py

```diff
@@ -1,7 +1,7 @@
 """Module for generating markdown strings for dagster ops"""
 from typing import Dict, List
 
-from dagster.core.definitions import NodeDefinition
+from dagster._core.definitions import NodeDefinition
```

Comment:
Based on the import path I would assume that `NodeDefinition` should not be imported.

Reply:
Right now, I don't see any other option